api/src/main/java/me/lokka30/treasury/api/common/event/EventTypeTracker.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/lokka30/Treasury>.
+ */
+
+package me.lokka30.treasury.api.common.event;
+
+import com.google.common.reflect.TypeToken;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+class EventTypeTracker {
+
+    private Map<Class<?>, List<Class<?>>> friends = new HashMap<>();
+
+    public List<Class<?>> getFriendsOf(Class<?> eventType) {
```

Comment:
missing javadoc

Reply:
thats an implementation detail ; we don't need javadoc